vms/platformvm/txs/executor/helpers_test.go

```diff
@@ -89,16 +79,30 @@ func newEnvironment(t *testing.T, postBanff, postCortina, postDurango bool) *env
 	var isBootstrapped utils.Atomic[bool]
 	isBootstrapped.Set(true)
 
-	config := defaultConfig(postBanff, postCortina, postDurango)
-	clk := defaultClock(postBanff || postCortina || postDurango)
+	var (
+		fork     configtest.ActiveFork
+		forkTime time.Time
+	)
+	switch {
+	case postDurango:
+		fork = configtest.DurangoFork
+		forkTime = genesistest.ValidateStartTime.Add(-2 * time.Second).Add(-2 * time.Second)
+	case postCortina:
+		fork = configtest.CortinaFork
+		forkTime = genesistest.ValidateStartTime.Add(-2 * time.Second).Add(-2 * time.Second)
+	case postBanff:
+		fork = configtest.BanffFork
+		forkTime = genesistest.ValidateEndTime.Add(-2 * time.Second)
+	default:
+		fork = configtest.ApricotPhase5Fork
+		forkTime = genesistest.GenesisTime
+	}
```

Comment:
this is just a refactoring of the way we set fork times right now. We'll fix this in next PR